Handling sensitive data in Umbraco Forms

eshanrnh opened the pull request:
## Description

Created a section "Handling sensitive data in Umbraco Forms" at the end of the Security article.

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [x] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

v10, 12, 13 and 14

## Deadline (if relevant)

Anytime

sofietoft reviewed (commented):
Looks great 💪 
I've added a couple of comments.

Does it make sense to add a bit more information about the feature in the first part?
Like, talking about the default Sensitive Data user group, explaining what it is and what it means to be a part of this group? Or perhaps it could be enough to link to the [article about this in the Umbraco CMS docs](https://docs.umbraco.com/umbraco-cms/reference/security/sensitive-data-on-members)? 🤔 

Also, it's a guide in the Reference section 😅 
I think this is by far the best place for this information, but... the over all structure isn't great 🤷‍♀️ Something to tackle at a later point I think 😁

eshanrnh commented:
Thanks for the review, @sofietoft 🙌 I've incorporated the comments and added a link to the CMS article.

I agree regarding the structure of the article but the entire section might need some revamping at some point of time 🙈

If these changes look fine, I'll make them for the rest of the versions too.

sofietoft reviewed (approved):
Looks great 💪
